openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1212,170 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def configure_setup(self, max_volume_count, total_volume_count):
+        # Get revised version of heketi
+        revision = oc_get_custom_resource(
+            self.ocp_master_node[0], 'dc', ':status.latestVersion',
+            self.heketi_dc_name)
+
+        # Set env variable to support max 2000 volumes in heketi dc
+        cmd = ("oc set env dc/{} HEKETI_GLUSTER_MAX_VOLUMES_PER_CLUSTER="
+               "{}".format(self.heketi_dc_name, max_volume_count))
+        self.cmd_run(cmd)
+
+        # Get new revised version of heketi
+        new_revision = oc_get_custom_resource(
+            self.ocp_master_node[0], 'dc', ':status.latestVersion',
+            self.heketi_dc_name)
+
+        # Wait for new pod to be up
+        if int(new_revision[0]) != int(revision[0]):
+            selector = 'deployment={}-{}'.format(
+                self.heketi_dc_name, new_revision[0])
+            for w in Waiter(300, 3):
+                if oc_get_custom_resource(
+                        self.ocp_master_node[0], "pod", ':.metadata.name',
+                        selector=selector)[0]:
+                    break
+
+            if w.expired:
+                err = 'Pod not found with label {}'.format(selector)
+                raise AssertionError(err)
```

Comment:
I would suggest directly useing the err msg,
raise AssertionError('Pod not found with label {}'.format(selector))